docs/src/main/tut/invariant.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: default
+title:  "Invariant"
+section: "typeclasses"
+source: "https://github.com/non/cats/blob/master/core/src/main/scala/cats/functor/Invariant.scala"
+scaladoc: "#cats.functor.Invariant"
+---
+# Invariant
+
+The `Invariant` typeclass is for `Functor`s that define an `imap`
```

Comment:
We should probably be careful with wording here. In Cats, `Functor` means covariant functor. To me, this sentence makes it sound like `Invariant` is a subtype of `Functor`, while it's actually the other way around. This may be solved by just changing `Functor` to a generic "functor". That could potentially also be confusing to people who think that `Functor` and "functor" would be synonymous. It may be fine though.